<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormReport700</class>
 <widget class="QWidget" name="FormReport700">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FormReport700{border-image: url(:/image/report.png);}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>481</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>140</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>31</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>41</number>
   </attribute>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>商品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>重量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>支付方式</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请选择上列报表类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_next">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prior">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-----</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>449</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-----</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_item">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>0</y>
     <width>201</width>
     <height>391</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>149</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>91</number>
   </attribute>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_print">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>419</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-----</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
